### PR TITLE
[@types/ibm_db] Fixed issue #60848

### PR DESCRIPTION
**Pull request by Stoyz**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/ibmdb/node-ibm_db/blob/master/APIDocumentation.md#-8-closecallback>>
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

---

Change based on [this comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60879#pullrequestreview-1017797399).

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ibm_db ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ibm_db@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'ibm_db'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4adb127c95e10f4785cd79bef6fbde71c2c1992d
</p>

File: types/ibm_db/index.d.ts [hunk 1]
```diff
@@ -63,8 +63,8 @@ export class Database implements Options {
 
     openSync(connStr: string | ConnStr): boolean;
 
-    close(cb?: (err: Error, db: Database) => any): void;
     close(): Promise<void>;
+    close(cb?: (err: Error, db: Database) => any): void;
```
[the remark this answers is not in the document]
Comment (peterblazejewicz) [reply types/ibm_db/index.d.ts:67]: you would have to modifyt this DT definition tests to incorporate one to test your change btw

[the remark this answers is not in the document]
Comment (peterblazejewicz) [reply types/ibm_db/index.d.ts:67]: Yes, sure. This is how review works. Please update tests to incorporate code closing connection with changed definition

**Comment by Stoyz** [comment]
Okay one thing I don't get is how the `ibm_db-tests.ts` file is supposed to contain tests? I don't see any kind of tests.
Could you please explain to me @peterblazejewicz ?

**Comment by Stoyz** [comment]
Okay I have updated the PR.
Thank you very much for your patience.

**Review by jakebailey — APPROVED** [review]

**Comment by Stoyz** [comment]
Ready to merge

**Pull request merged**